chia/full_node/full_node.py

```diff
@@ -1693,7 +1697,7 @@ async def get_sub_slot_iters_difficulty_ses_block(
         assert diff is not None
         return ssi, diff, prev_ses_block
 
-    async def _finish_sync(self) -> None:
+    async def _finish_sync(self, fork_point: Optional[int]) -> None:
```

Comment:
right, this was because we are doing some manipulation inside finish sync that could result in -1 and use the same var, we later use max(fork,o) to make sure we put a uint32 inside StateChangeSummary, i refactored so this could just be uint32 all the way